.github/workflows/deploy-docker.yaml

```diff
@@ -109,48 +119,30 @@ jobs:
         with:
           context: .
           file: apps/${{ matrix.packages.name }}/Dockerfile
-          tags: |
-            ${{ inputs.container-registry }}/${{ inputs.image-prefix }}/${{ matrix.packages.name }}:${{ matrix.packages.imageTag }}
+          tags: ${{ inputs.container-registry }}/${{ inputs.image-prefix }}/${{ matrix.packages.name }}:${{ matrix.packages.imageTag }}
           push: ${{ inputs.push }}
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new
 
-  # update-deployment-declaration:
-  #   name: Update deployment declaration
-
-  #   if: ${{ inputs.update && needs.build-and-publish-docker-image.outputs.DOCKERFILE_EXISTS == 'true' }}
-
-  #   needs:
-  #     - build-and-publish-docker-image
-
-  #   runs-on: ubuntu-latest
-
-  #   steps:
-  #     - name: Checkout Repo
-  #       uses: actions/checkout@v3
-  #       with:
-  #         repository: ${{ inputs.target-repository }}
-  #         ref: ${{ inputs.target-ref }}
-
-  #     - name: Update publishPackages version in YAML file
-  #       run: |
-  #         for row in $(echo "$PACKAGES" | jq -r '.[] | @base64'); do
-  #           _jq() {
-  #             echo ${row} | base64 --decode | jq -r ${1}
-  #           }
-  #           NAME=$(_jq '.name')
-  #           VERSION=$(_jq '.version')
-  #           OLD_VERSION=${{ inputs.container-registry }}\/${{ inputs.image-prefix }}\/$NAME:.*
-  #           NEW_VERSION=${{ inputs.container-registry }}\/${{ inputs.image-prefix }}\/$NAME:$VERSION
-  #           find ./${{ inputs.target-directory }} -type f -exec sed -i -e "s|$OLD_VERSION|$NEW_VERSION|g" {} \;
-  #           echo "${NAME}:${VERSION} is updated"
-  #         done
-  #       env:
-  #         PACKAGES: ${{ inputs.packages }}
-
-  #     - name: Show git status
-  #       run: git status
-
-  #     - name: Open Pull Request to the target repository
-  #       uses: peter-evans/create-pull-request@v4
-  #       with:
-  #         title: Update Package versions
-  #         commit-message: Update Package versions
+      - name: Move Docker cache
+        if: steps.check-dockerfile.outputs.DOCKERFILE_EXISTS == 'true'
+        run: |
+          rm -rf /tmp/.buildx-cache
+          mv /tmp/.buildx-cache-new /tmp/.buildx-cache
+
+  update-gitops:
```

Comment:
Should we move this to the release workflows and keep this one for build only?

Reply:
If that so, the `release.yaml` will have double `update-gitops.yaml` since GitHib action has no `OR` needs. Also, we may update GitOps if and only if we build docker. I think it make sense to put this here.